test-it/karma/debugpoint/DebugPoint.Test.js

```diff
@@ -13,38 +13,67 @@ describe('A via file with debug point ', function () {
+            expect(vireo.eggShell.testNeedsUpdateAndReset(resultLocal)).toBeFalse();
+            const {rawPrint, rawPrintError} = await runSlicesAsync();
+            expect(rawPrint).toBeEmptyString();
+            expect(rawPrintError).toBeEmptyString();
+            expect(vireo.eggShell.testNeedsUpdateAndReset(resultLocal)).toBeTrue();
+        });
 
-    it('on executing succesfully set the needs update property on corresponding local ', async function () {
-        const runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, simpleAddNodeViaUrl);
-        const sumIndicator = vireo.eggShell.findValueRef('MyVI', 'dataItem_Sum');
-        expect(vireo.eggShell.testNeedsUpdateAndReset(sumIndicator)).toBeFalse();
-        const {rawPrint, rawPrintError} = await runSlicesAsync();
-        expect(rawPrint).toBeEmptyString();
-        expect(rawPrintError).toBeEmptyString();
-        expect(vireo.eggShell.testNeedsUpdateAndReset(sumIndicator)).toBeTrue();
+        it('on not executing will not set the needs update property on corresponding local', async function () {
+            const runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, addNodeWithCaseStructureViaUrl);
+            const resultLocal = vireo.eggShell.findValueRef('MyVI', 'local7');
+            expect(vireo.eggShell.testNeedsUpdateAndReset(resultLocal)).toBeFalse();
+            const {rawPrint, rawPrintError} = await runSlicesAsync();
+            expect(rawPrint).toBeEmptyString();
+            expect(rawPrintError).toBeEmptyString();
+            expect(vireo.eggShell.testNeedsUpdateAndReset(resultLocal)).toBeFalse();
+        });
     });
 
-    it('on not executing will not set the needs update property on corresponding local', async function () {
-        const runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, addNodeWithCaseStructureViaUrl);
-        const sumIndicator = vireo.eggShell.findValueRef('MyVI', 'dataItem_Sum');
-        expect(vireo.eggShell.testNeedsUpdateAndReset(sumIndicator)).toBeFalse();
-        const {rawPrint, rawPrintError} = await runSlicesAsync();
-        expect(rawPrint).toBeEmptyString();
-        expect(rawPrintError).toBeEmptyString();
-        expect(vireo.eggShell.testNeedsUpdateAndReset(sumIndicator)).toBeFalse();
+    describe('and debugging disabled, ', function () {
+        beforeAll(function (done) {
+            fixtures.preloadAbsoluteUrls([
+                simpleAddNodeViaUrl,
+                addNodeWithCaseStructureViaUrl
+            ], done);
+        });
+
+        beforeAll(async function () {
+            vireo = await vireoHelpers.createInstance();
+        });
+
+        afterAll(function () {
+            vireo = undefined;
+        });
+
+        it('on executing succesfully will not set the needs update property on corresponding local ', async function () {
+            const runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, simpleAddNodeViaUrl, true);
```

Comment:
Based on the other feedback this becomes

```js
const runSlicesAsync = vireoRunner.rebootAndLoadVia(vireo, simpleAddNodeViaUrl, {debugging: true});
```

Reply:
Done